### ISSUE TITLE
Troubleshooting Webhooks

### ISSUE THREAD

**Issue by pamgoodrich** [stoplightio/platform-docs#246]
1. You can use this feature to reanalyze a branch. I used this when a webhook failed to publish docs to platform.

https://meta.stoplight.io/docs/platform/ZG9jOjQ2NDU5Mw-branch-management#track-branches

Should go here as a troubleshooting step:

https://meta.stoplight.io/docs/platform/ZG9jOjM1NzUwMjEw-troubleshoot-git

2. Also, Chelsea asked that we add this to the track branches content...

and all branches associated with a pull request will be automatically tracked for you
"if the Stoplight webhook is installed"

3. Change the Troubleshoot Webhooks link in this topic to point to the Troubleshoot Git topic. https://meta.stoplight.io/docs/platform/ZG9jOjM0MjE2NTA5-git-projects

4. Fix typo in this section: https://meta.stoplight.io/docs/platform/ZG9jOjQ2NDU5Mw-branch-management#track-branches (ssocicated > associated)

5. Add recloning your repository to the Troubleshooting Git topic. I'd probably put it below the "Troubleshoot Webhooks" section but above "Reset Stoplight Access."

Steps: 
1. Edit a project in Studio Web.
2. Make sure all changes are pushed to the Git server.
3. From the Settings/hamburger menu, select Reclone Project.

Basically, this removes your local version of the branch and replaces it with the server version.

**Comment by pamgoodrich** [comment]
@sydstoplight : I added two more changes for this issue (#4 and #5). Let me know if you have questions.
